spec/locales-currencies-tz.html

```diff
@@ -54,13 +54,44 @@ <h1>IsStructurallyValidLanguageTag ( _locale_ )</h1>
       </p>
     </emu-clause>
 
-    <emu-clause id="sec-canonicalizelanguagetag" aoid="CanonicalizeLanguageTag">
```

Comment:
There are references to this now changed ESID in `annexes.html`, `locales-currencies-tz.html`, `negotiation.html` that need to be updated.